client/src/components/DataFiles/DataFilesStatus/DataFilesStatus.js

```diff
@@ -40,7 +40,13 @@ const OPERATION_MAP = {
           .join('/');
         const dest =
           destPath === '/' || destPath === ''
-            ? `${findSystemDisplayName(systemList, response.systemId)}/`
+            ? `${findSystemOrProjectDisplayName(
```

Comment:
@Strmiska , this is the main proposed change in the PR. 

the idea of deducing if the scheme is 'projects' by looking at the systemId sounds good to me 👍 

I think it need to be adjusted a bit as it is possible that the project system id's looks different as they depend on a configurable `_PORTAL_DATA_DEPOT_PROJECT_SYSTEM_PREFIX` in the backend.   So its usually `something.project.something` but it could be something different like `something.PROJECT.something` or `something.shared_workspace.something`.

So my initial suggestion would be to provide _PORTAL_DATA_DEPOT_PROJECT_SYSTEM_PREFIX to the frontend so that it could be used here.

Reply:
Thanks Nathan, I think this is more of a solid solution! I'll get this and JoonYee's changes pushed up soon.